cpmonitor/templatetags/admin_task_tree.py

```diff
@@ -0,0 +1,10 @@
+from treebeard.templatetags.admin_tree import result_tree as tb_result_tree
+from treebeard.templatetags.admin_tree import register
+
+
+@register.inclusion_tag("admin/tree_change_list_results.html", takes_context=True)
+def result_tree(context, cl, request):
+    """ """
```

Comment:
Good idea to add some docs here, but it could be expanded a little ;)

Jokes aside, I'd find docs helpful here to understand why we apparently remove the filtering from these results.